.github/workflows/deploy_preview.yml

```diff
@@ -22,13 +28,33 @@ jobs:
         run: npm ci
       - name: Build docs preview
         run: npm run build:docs:preview
+      - name: Download VRT reports (All flags enabled)
+        uses: actions/download-artifact@v4
+        with:
+          name: vrt-all-flags
+          path: docs/public/vrt-all-flags
+      - name: Download VRT reports (No flags enabled)
+        uses: actions/download-artifact@v4
+        with:
+          name: vrt-no-flag
+          path: docs/public/vrt-no-flag
+      - name: Download AAT reports (All flags enabled)
+        uses: actions/download-artifact@v4
+        with:
+          name: axe-all-flags
+          path: docs/public/aat-all-flags
+      - name: Download AAT reports (No flags enabled)
+        uses: actions/download-artifact@v4
+        with:
+          name: axe
+          path: docs/public/aat-no-flag
```

Comment:
For `actions/download-artifact`, would we need to pass along `run-id` to these in order to download from the CI workflow? My assumption is that these try to download from the current workflow but let me know if that's incorrect

Reply:
That was also my assumption and they seem to be updating correctly on each run